sdcm/sct_events/setup.py

```diff
@@ -131,6 +131,16 @@ def enable_default_filters(sct_config: SCTConfiguration):  # pylint: disable=unu
                                 event_class=CassandraStressLogEvent.ConsistencyError,
                                 regex=r".*Authentication error on host.*Cannot achieve consistency level for cl ONE").publish()
 
+    if sct_config.get('new_scylla_repo') or sct_config.get('new_version'):
+        # scylladb/scylla-enterprise#3814
+        # scylladb/scylla-enterprise#3092
+        # skip audit related issue when upgrading from older versions it's default in
+        # TODO: remove when branch 2022.2 is deprecated
+        EventsSeverityChangerFilter(new_severity=Severity.WARNING,
```

Comment:
why warning instead of 'debug'/'info'?

Reply:
same as all of the above, that move it to warning.

I guess the initial idea was that those can be found easily if needed, but it doesn't really make any difference